tmk_core/common/report.h

```diff
@@ -190,9 +190,14 @@ typedef struct {
     int8_t  h;
 } __attribute__((packed)) report_mouse_t;
 
+
 typedef struct {
 #if JOYSTICK_AXES_COUNT > 0
-    int8_t axes[JOYSTICK_AXES_COUNT];
+#    if JOYSTICK_AXES_RESOLUTION <= 8
+        int8_t axes[JOYSTICK_AXES_COUNT];
+#    else
+        int16_t axes[JOYSTICK_AXES_COUNT];
+#    endif
```

Comment:
```suggestion
#    if JOYSTICK_AXES_RESOLUTION > 8
        int16_t axes[JOYSTICK_AXES_COUNT];
#    else
        int8_t axes[JOYSTICK_AXES_COUNT];
#    endif
```

Reply:
This looks good, I have added this and removed the newlines